apps/admin/src/resources/users/tabs/permissions-tab.tsx

```
@@ -205,6 +252,32 @@ function AddPermissionButton() {
                 </SelectContent>
               </Select>
             </div>
+            {orgsLoadFailed && (
+              <p className="text-sm text-destructive">
+                Couldn't load organizations. Retry before adding a permission —
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&`#39`;`, `&rsquo;`.

(react/no-unescaped-entities)